### PR TITLE
Add aria2 (aria2c) to Windows

### LINKED ISSUE #3670: Add aria2 on windows
### Tool name

aria2

### Tool license

GPL 2

### Add or update?

- [X] Add
- [ ] Update

### Desired version

1.35.0

### Approximate size

2MB

### Brief description of tool

aria2 is already available on linux and on macOS

aria2 - The ultra fast download utility

aria2 is a utility for downloading files. The supported protocols are HTTP(S), FTP, SFTP, BitTorrent, and Metalink. aria2 can download a file from multiple sources/protocols and tries to utilize your maximum download bandwidth. It supports downloading a file from HTTP(S)/FTP/SFTP and BitTorrent at the same time, while the data downloaded from HTTP(S)/FTP/SFTP is uploaded to the BitTorrent swarm. Using Metalink's chunk checksums, aria2 automatically validates chunks of data while downloading a file like BitTorrent.

### URL for tool's homepage

https://github.com/aria2/aria2

### Provide a basic test case to validate the tool's functionality.

```bash
aria2c --version
```

### Virtual environments affected

- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11
- [X] Windows Server 2016
- [X] Windows Server 2019

### Can this tool be installed during the build?

```bash
choco install
```

### Tool installation time in runtime

1s

### Are you willing to submit a PR?

Yes

### PR DESCRIPTION
**Pull request by jmarrec**
# Description

Fix #3670 - Add aria2 (aria2c) to Windows

Respawn of  #3671 since that one was closed.

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-Aria2.ps1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+################################################################################
```
Comment (al-cheb) [inline images/win/scripts/Installers/Install-Aria2.ps1:1]: Could you please remove this script and update toolset-2016.json/2019.json files?

File: images/win/windows2016.json [hunk 2]
```diff
@@ -312,7 +312,8 @@
                 "{{ template_dir }}/scripts/Installers/Install-Vcpkg.ps1",
                 "{{ template_dir }}/scripts/Installers/Install-MongoDB.ps1",
                 "{{ template_dir }}/scripts/Installers/Install-GoogleCloudSDK.ps1",
-                "{{ template_dir }}/scripts/Installers/Install-CodeQLBundle.ps1"
+                "{{ template_dir }}/scripts/Installers/Install-CodeQLBundle.ps1",
```
Comment (al-cheb) [inline images/win/windows2016.json:315]: Please remove these changes.

File: images/win/scripts/Installers/Install-Aria2.ps1 (hunk 1 above)
Comment (jmarrec) [reply images/win/scripts/Installers/Install-Aria2.ps1]: That didn't exist three weeks ago. I made the changes you requested. Could you take a look again please @al-cheb ?

**Review by al-cheb — APPROVED** [review]

**Comment by AlenaSviridenko** [comment]
Hi @jmarrec, please, pull the latest main branch to get the latest fix for Windows images. Thanks.

**Comment by jmarrec** [comment]
@AlenaSviridenko done. 

For the record I allow all maintainers of this repo to push changes to my fork, so feel free to pull main into it as you see fit.

image

**Review by miketimofeev — APPROVED** [review]

**Comment by jmarrec** [comment]
Thanks for the merge!

**Comment by miketimofeev** [comment]
@jmarrec thanks for the contribution! The new images with aria2 will be available next week.

**Pull request merged**